tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+Once you have an [Events dataset][events], you can create a web application that captures session events.
+
+This example shows how you can send log information via Flume to your Hadoop database using a JSP and custom servlet running on Tomcat.
```

Comment:
This is great context, but I'd like to have a bit more information. At least:

* That you can use the generate-events tutorial as an alternative step if you want to move on and don't care about Flume.
* That these events will be sessionized in the next tutorial. This is to give a little more _why_ context.
* That the JSP and custom servlet running on Tomcat are not important. Those components are in place of the user's real application that has interesting events to look at and sessionize.